src/core/Browser.js

```diff
@@ -75,7 +75,7 @@ export var gecko3d = 'MozPerspective' in style;
 
 // @property any3d: Boolean
 // `true` for all browsers supporting CSS transforms.
-export var any3d = !window.L_DISABLE_3D && (ie3d || webkit3d || gecko3d) && !opera12 && !phantom;
+export var any3d = !window.L_DISABLE_3D && (ie3d || webkit3d || gecko3d || (CSS && CSS.supports('transform', 'translate3d(0,0,0)'))) && !opera12 && !phantom;

```

Comment:
Is is worth removing phantom and operator from here as these are end-of-life (I think)